test/regression/cases/uds_dogstatsd_to_api_cpu/experiment.yaml

```diff
@@ -1,21 +1,27 @@
 optimization_goal: cpu
 erratic: false
 
-environment:
-  DD_TELEMETRY_ENABLED: true
-  DD_API_KEY: 00000001
+target:
```

Comment:
The fact that we have to repeat this so many times makes me wonder if it should be moved into `config.yaml`.

This would restrict a given `regression` folder to a single target, which seems reasonable to me, but could be restrictive for some other repos (ie, dogfooding)

Don't think it needs to be addressed now, just food for thought.

Reply:
I think the logic behind moving it into the experiment.yaml file is that it creates the flexibility later to make the name and command a per experiment vs per job parameter. Just like how the cpu and memory allotment will eventually get moved inside as well.